app/routes/routes.py

```diff
@@ -0,0 +1,86 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
+
+# class Planet():
+#     def __init__(self, id, name, description, color):
+#         self.id = id
+#         self.name = name
+#         self.description = description
+#         self.color = color
+
+# PLANETS = [
+#     Planet(1, "Saturn", "gassy giant", "orange"),
+#     Planet(2, "Mercury", "small rocky", "gray"),
+#     Planet(3, "Pluto", "still a planet", "white")
+# ]
+
+planet_bp = Blueprint("Planet", __name__, url_prefix="/planet")
+
+@planet_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
```

Comment:
Doing some validation on the `request_body` to ensure it has the required fields would be good here.